lib/KTextbox/index.vue

```diff
@@ -164,6 +164,13 @@
         type: Boolean,
         default: false,
       },
+      /**
+       * CSS styles to apply to the container
+       */
+      containerAppearanceOverrides: {
```

Comment:
I think we could go just with usual `appearanceOverrides`.

Since I wouldn't expect we will need to override two inner elements, and have need for another kind of `appearanceOverrides` (and if if we needed to override two or more inner elements, that'd be time to think about how to architecture the component better).

Reply:
Thanks! Will go with this!